src/Microsoft.Docs.Template/Models/TableOfContents.cs

```diff
@@ -20,8 +20,16 @@ public class TableOfContentsMetadata
     {
         public List<string> Monikers { get; set; }
 
+        [JsonProperty(PropertyName = "monikerRange")]
+        public string MonikerRange { get; set; }
+
         [JsonExtensionData]
         public JObject ExtensionData { get; set; }
+
+        public bool ShouldSerializeMonikerRange()
```

Comment:
what is this for?

Reply:
to ignore monikerRange when serialization